frontend/src/Dashboard/index.js

```diff
@@ -17,77 +17,83 @@ export const Dashboard = props => {
   const [activePage, changePage] = useState(1);
   const [classesOnPage, setCurrentClasses] = useState([]);
 
-  const fetchClasses = () => {
+  const fetchClasses = useCallback(() => {
     getClassRequestor(localStorage.getItem('userid'))
       .then(response => {
         // Sets the state to the data from the classes endpoint (class[])
         setClasses(response.data);
-        setCurrentClasses(response.data.slice((activePage-1)*10, activePage*10));
+        setCurrentClasses(
+          response.data.slice((activePage - 1) * 10, activePage * 10),
+        );
         setLoading(false);
       })
       .catch(error => {
         // Error handler
         setLoading(false);
         console.error(error);
       });
-  }
+  });
 
   useEffect(() => {
     setLoading(true);
     fetchClasses();
-  }, []);
+  }, [fetchClasses]);
```

Comment:
Why is this here?

Reply:
Linter is throwing errors. Not sure who wrote that code but they should change it.